src/cljs/coffeepot/localization.cljs

```diff
@@ -0,0 +1,11 @@
+(ns coffeepot.localization
+    (:require [tongue.core :as tongue]))
+
+(def locales {
+    :login {
+        :en "Login with {1}"
+        :fi "Kirjaudu sisään {1}}:lla"}

```

Comment:
Onko tuo yksi sulkeva aaltosulje ylimääräinen?

Reply:
It would appear so sir!